Automattic/woocommerce-payments
# feat: update account data on upgrade

Fixes https://github.com/Automattic/woocommerce-payments/issues/2520

#### Changes proposed in this Pull Request









#### Testing instructions





- Check out this branch
- Have the server started locally
- On the server, add `add_filter( 'wcpay_feature_include_apm_fees_in_account_fee_info', '__return_true' );` somewhere - e.g. in a mu-plugin
- Ensure you have the "Enable UPE checkout" flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- You should see the "missing fees" message
- In your DB, update the `woocommerce_woocommerce_payments_version` to a previous version, like `2.6.0`
```sql
UPDATE `wp_options` SET `option_value` = '2.6.0' WHERE `wp_options`.`option_name` = 'woocommerce_woocommerce_payments_version';
```
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- You should now see the fees

-------------------
